api/client/lib.py

```diff
@@ -598,14 +598,19 @@ def get_geo_centre(access_token, api_host, region_id):
 
 
 @memoize(maxsize=None)
-def get_geojson(access_token, api_host, region_id):
-    url = '/'.join(['https:', '', api_host, 'v2/geocentres?includeGeojson=True&regionIds=' +
-                    str(region_id)])
+def get_geojsons(access_token, api_host, region_id, descendant_region_level_id=None):
+    url = '/'.join(['https:', '', api_host, 'v2/geocentres?includeGeojson=True&regionIds={}'.format(
+        region_id)])
+    if descendant_region_level_id:
+        url += "&reqRegionLevelId={}&stringify=false".format(descendant_region_level_id)
     headers = {'authorization': 'Bearer ' + access_token}
     resp = get_data(url, headers)
-    for region in resp.json()['data']:
+    return [dict_reformat_keys(r, str_camel_to_snake) for r in resp.json()['data']]
+
+
+def get_geojson(access_token, api_host, region_id):
```

Comment:
Memoize decorator on this one as well?

Reply:
not memoizing here was intentional, since this fn calls the other ,there's no benefit memoizing the middle man, and the cost is 2x the ram

Reply:
Ah I see. 👍